pkgs/development/libraries/vtk/vtk.egg-info

```diff
@@ -0,0 +1,4 @@
+Metadata-Version: 2.1
+Version: @VTK_VER@
+Summary: VTK is an open-source toolkit for 3D computer graphics, image processing, and visualization
+Platform: UNKNOWN
```

Comment:
Is this field important? We could fill it with nix.

Reply:
I doubt it; regardless `UNKNOWN` is the value I have seen in other `egg-info` files in the wild so I would feel better not deviating.